common.go

```diff
@@ -47,9 +47,7 @@ func NewString(value string) *string {
 // Bool is a helper routine that allocates a new bool value
 // to store v and returns a pointer to it.
 func Bool(v bool) *bool {
-	p := new(bool)
-	*p = v
-	return p
+	return &v
```

Comment:
I'm confused about this change and the function in general, doesn't this have a different result than the comment above this function?

Reply:
@taimoor-at-opslevel  So the problem is that you cannot do this
```
input := ScorecardInput{
  AffectsOverallServiceLevels: &true
}
```

So you have two ways to fix it - pass the bool value through a function to get it stored in a var and then return the address of the variable 
```
input := ScorecardInput{
  AffectsOverallServiceLevels: ol.Bool(true)
}
```
OR make a local var and give the address
```
myBool := true
input := ScorecardInput{
  AffectsOverallServiceLevels: &myBool
}
```

Reply:
The way the function was originally written was unnecessary - its creates an unneeded pointer - the way David fixed it is more idomatic go